stories/toast.stories.js

```diff
@@ -297,6 +293,8 @@ export const Usage = () => {
           button={buttonP}
           icon={<CheckBadge size='lg'/>} 
           onClose={() => setPopUp(false)}
+          timeOut={5000}
+          onTimeOut={() => setPopUp(false)}
```

Comment:
`setPopUp => false` here is already called within the underlying `timeout` function so wouldn't need it here right, see above comment ^^